Library/Formula/gst-python.rb

```diff
@@ -0,0 +1,22 @@
+class GstPython < Formula
+  homepage "http://gstreamer.freedesktop.org/modules/gst-python.html"
+  url "http://gstreamer.freedesktop.org/src/gst-python/gst-python-1.4.0.tar.xz"
+  sha256 "b1e40c29ceb41b03f08d38aca6056054f0341d0706276326dceeec6ac8d53d3e"
+
+  head "http:/"
```

Comment:
Is the new commit what you expected? http://gstreamer.freedesktop.org/src/gst-python/gst-python-1.4.0.tar.xz